cypress/support/fragments/data_import/mapping_profiles/newFieldMappingProfile.js

```diff
@@ -14,7 +14,6 @@ import getRandomPostfix from '../../../utils/stringTools';
 
 const saveButton = Button('Save as profile & Close');
 const organizationModal = Modal('Select Organization');
-const adminNoteFieldName = 'profile.mappingDetails.mappingFields[9].repeatableFieldAction';
 const marcBib = 'MARC Bibliographic';
```

Comment:
'marcBib' is assigned a value but never used. Allowed unused vars must match /React/u

it seems to me that you initially created the variable "marcBib" in order to pass its value to the properties of the "incomingRecordType" and "folioRecordTypeValue" objects, but you did not do this, so please delete the variable if you do not need it